### PR TITLE
feat(FX-3926): display Sort & filter bar when artist has no artworks

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves [FX-3926]

### Acceptance Criteria
* Display create alert/sort & filter bar when artist has no artworks
* Change empty artwork state copy to match the copy in [Figma](https://www.figma.com/file/zG9cduJdOaTQAw886QcOwn/Filters?node-id=3916%3A35344)

#### Video
https://user-images.githubusercontent.com/3513494/169790743-991a10fe-02b2-4da1-804a-6eac9e6242bc.mp4

#### Screenshot
Simulator Screen Shot - iPhone 8 - 2022-05-23 at 12 37 25

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [x] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- display Sort & filter bar when artist has no artworks - dimatretyak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[FX-3926]: https://artsyproduct.atlassian.net/browse/FX-3926?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/Artist/ArtistArtworks/ArtistArtworksEmptyMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+export const ArtistArtworksEmptyMessage = () => {
+  return (
+    <Flex p={2} bg="black5">
+      <Text variant="sm" color="black100">
+        No works available by the artist at this time
+      </Text>
+      <Text variant="sm" mt={0.5} color="black60">
+        Create an Alert to receive notifications when new works are added
+      </Text>
+    </Flex>
```
Comment (dimatretyak) [inline src/app/Components/Artist/ArtistArtworks/ArtistArtworksEmptyMessage.tsx:12]: [Asked a question](https://artsyproduct.atlassian.net/browse/FX-3926?focusedCommentId=48771) about whether we should use the `Message` component here

**Review by gkartalis — COMMENTED** [review]
Checking also on the emulator now

File: src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx [hunk 2]
```diff
@@ -202,7 +201,16 @@ const ArtistArtworksContainer: React.FC<ArtworksGridProps & ArtistArtworksContai
...
-    return <ArtistArtworksEmptyMessage />
+    return (
+      <Message
+        variant="default"
+        title="No works available by the artist at this time"
+        text="Create an Alert to receive notifications when new works are added"
+        bodyTextStyle={{
+          color: "black60",
+        }}
+      />
```
Comment (gkartalis) [inline src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx:212]: 👏 Thanks!

**Review by gkartalis — APPROVED** [review]

File: src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx (hunk 2 above)
Comment (dimatretyak) [reply src/app/Components/Artist/ArtistArtworks/ArtistArtworks.tsx:212]: thank you for investigating this moment

**Pull request merged**